<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBudget</class>
 <widget class="QDialog" name="EditBudget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Account</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Summ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="AccountOper2" name="accountEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="currencyEdit" name="summEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Other credits</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="monthEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccountOper2</class>
   <extends>QLineEdit</extends>
   <header location="global">accountoper2.h</header>
  </customwidget>
  <customwidget>
   <class>currencyEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/currencyedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
